### PR TITLE
Dev Workflow article.

### PR DESCRIPTION
**Pull request by reynoldsalec**
Adapted this from an email I wrote...doesn't match our personas that well but wanted to have it as a reference to give to people like the folks I was talking with.

### CODE REVIEW

File: src/content/articles/agency-dev-workflow-2019.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+img-src: https://thinktandem.io/images/articles/linter-is-coming/linteriscoming.jpeg
+byline: Don't get caught wearing last year's dev workflow, see all the essentials for 2019.
+date: 2019-12-20
+---
+
+## SVN is so 2009.
+
+Your developers use SFTP because it's the fastest way to edit a site. Version control seems like a good idea, but it's been hard to find the right workflow that doesn't require shelling out money for Github accounts, managed hosting providers, and other accessories. You've heard of Docker and containers, but it's hard to think about moving off of hosting that's worked for years.
+
+As the creators of [Lando](https://www.devwithlando.io) and organizers of the Bay Area Drupal Camp DevOps Summit, we often hear from teams like yours. Often they're a bit afraid of being left behind, but also curious to see how other teams are handling the process of writing, testing, and deploying code.
```
Comment (serundeputy) [inline src/content/articles/agency-dev-workflow-2019.md]: ~~https://www.devwithlando.io~~ --> https://docs.devwithlando.io

File: src/content/articles/agency-dev-workflow-2019.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Put all projects under Git version control and use that for deploying code changes. (Optional: Adopt Github as the central repository for better team-collaboration features like pull requests/issue management linked to code commits/etc.)
+
+### 2. Configuration Management
+
+Adopt an "all in code" approach on as many projects as practical. Modern professional CMSs should have configuration management tools available to do this; Drupal 7 had Features, WordPress has WP-CFM, and Drupal 8 has configuration management as a core feature.
+
+### 3. Local Development
+
+Use a Docker-based local development solution like [Lando](https://www.devwithlando.io). Your local development environment should support all the operating systems (Mac/Linux/Windows) and be able to adapt to all the application frameworks you create projects with (Drupal/WordPress/Laravel/NodeJS/etc.), but be simple enough that any new developer can get working within a couple minutes. Integration with professional hosting providers (see below) is a big plus.
```
Comment (serundeputy) [inline src/content/articles/agency-dev-workflow-2019.md]: ~~https://www.devwithlando.io~~ --> https://docs.devwithlando.io

File: src/content/articles/agency-dev-workflow-2019.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+- **Highly dynamic scalability:** Does the system use modern container architecture and reverse proxies to handle large amounts of traffic easily?
+
+We've used Pantheon and Platform.sh in the past, but many other good providers have these features.
+
+Often we hear a cost-based objection: "Our hosting costs $5/month, why would we switch to something 5 or 10 times as expensive?" If you're worried about cost, consider calculating the _true_ cost of your current hosting. If it can save you as little as half an hour a month, a professional hosting service will save money. Try starting with larger projects that likely require these features and moving to smaller ones.
+
+## Your First Accessories
+
+Once you've adopted these components, most teams are curious to hear about this magical "automated testing" they've been hearing about. It's such a common topicI couldn't resist adding a few thoughts!
```
Comment (serundeputy) [inline src/content/articles/agency-dev-workflow-2019.md]: ~~topicI~~ --> topic  I

**Comment by dbungard** [comment]
My only thought would be to include both internal links (seo to services/ a11y to that page etc) as well as outgoing links to platform & pantheon and the other tools.

**Comment by reynoldsalec** [comment]
Ok, finished writing some new section, added an image, and addressed @serundeputy's edits.

@dbungard let me know where else I could add some internal links (better yet you could [edit the file directly](https://github.com/thinktandem/thinktandem.io/edit/dev-workflow/src/content/articles/agency-dev-workflow-2019.md]).

**Pull request merged**